### ISSUE TITLE
[Feature Request] Uploading the specified file types

### ISSUE THREAD

**Issue by shawnpoo** [seebin/obsidian-minio-uploader-plugin#4]
Nice plugin! 

One feature required: on the Plugin Setting page, the specified file types can be filtered. So that the user can decide which file type to be uploaded.

Thanks!

**Comment by turnercore** [comment]
I agree with this, I may see if I can add it myself. I want to be able decide which filetypes to upload and which to ignore and let Obsidian handle. I'd also like to have the option of a prompt, so it would be 'always' 'prompt' or 'never' upload the specified filetype.
